eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1670,7 +1696,7 @@ protected function copyTranslationsFromPublishedVersion(APIVersionInfo $versionI
             }
         }
 
-        $this->updateContent($versionInfo, $updateStruct);
+        $this->internalUpdateContent($versionInfo, $updateStruct);
```

Comment:
This change requires `copyTranslationsFromPublishedVersion` PHPDoc to be updated.